src/app/pages/home/footer/footer.component.html

```diff
@@ -1 +1,31 @@
-<div>Footer Page</div>
\ No newline at end of file
+
+<nav class=" navbar navbar-dark bg-dark mt-10 clearfix">
+
+    <div class="navbar-expand m-auto navbar-text">
+        Made with
+        <fa-icon [icon] = "heart" class = "hrt" ></fa-icon>

```

Comment:
I would think having in-line styling would be perfect for one-off cases like this. Also note that you can just refer to the :heart: icon using the icon's styling prefix (`fas`) followed by the icon name. You can easily look for em' [here](https://fontawesome.com/icons/heart?style=solid).

```suggestion
        <fa-icon [icon]="['fas', 'heart']" style="color: red;"></fa-icon>   
```